cv19ResSupportV3.Tests/V3/UseCase/CreateHelpRequestCallUseCaseTest.cs

```diff
@@ -27,23 +28,19 @@ public void SetUp()
         public void ExecuteWithValidIdSavesRequestToDatabase()
```

Comment:
I don't think the test name reflects what's being tested.
When it comes to such usecase, two things can be tested here:
1) It returns what we expect it to return _(which seems to be what the assertion is implying)_
2) Usecase does whatever we expect it to do _(apart from returning result)_. (Which is what is being implied by the test name somewhat... well the test name says "Usecase Saves to DB", which is not it's responsibility, but... we could say instead `"ExecuteWithValidIdInvokesAGatewayWithExpectedParameters"`).
In this case, we should test whether the usecase calls the gateway with expected parameters:
`_mockHelpRequestCallGateway.Verify(g => g.CreateHelpRequestCall(It.Is<CreateHelpRequestCall>(c => {assertions on the command object})), Times.Once)`